microraiden/microraiden/channel_manager/blockchain.py

```diff
@@ -138,7 +138,7 @@ def _update(self):
             **filters_unconfirmed
         )
         for log in logs:
-            assert log['args']['_receiver'] == self.cm.state.receiver
+            assert log['args']['_receiver'].lower() == self.cm.state.receiver
```

Comment:
Suggestion: use `is_same_address` from `eth_utils` or `assert log['args']['_receiver'].lower() == self.cm.state.receiver.lower()` in case `cm.state.receiver` is not normalized (future-proofed)

Reply:
thanks, it definitely needs some polishing.
BTW I've used your `LogFilter`, definitely helped a lot!

Reply:
when everything is working I'll merge your PR #311 to this, do you agree?

Reply:
yes, it is better to have it in one place.